lib/access/model.rb

```diff
@@ -0,0 +1,31 @@
+require_relative "../base.rb"
+module Figo
+# Object representing one access of the User
+  class Access < Base
+    @dump_attributes = []
+
+    def initialize(session, json)
```

Comment:
🤔I think that this is unnecessary

Reply:
All models are written with the initialize. 
Should I leave it this way for coherence or change it?

Reply:
Since this endpoint is not testable for now, I'll leave it as it is until the API is fixed